template-manager-blueprint/src/main/resources/definitions/cloud-storage-location-specification.json

```diff
@@ -3,140 +3,110 @@
     {
       "propertyName": "logfeeder.cloud.storage.base.path",
       "propertyFile": "logfeeder-properties",
-      "description": "This is the directory where cluster daemon logs should be stored by Log Feeder",
+      "description": "This is the directory where cluster daemon logs will be stored by Log Feeder",
       "defaultPath": "{{{ defaultPath }}}/clusterlogs",
```

Comment:
I know I screwed this up but could you please insert `/oplogs` before `/clusterlogs`?

Reply:
Perfect, thanks!